samples/bluetooth/st_ble_sensor/README.rst

```diff
@@ -0,0 +1,59 @@
+.. _bluetooth-st_ble_sensor:
+
+Bluetooth: ST BLE Sensor Demo
+#############################
+
+Overview
+========
+This application demonstrates BLE peripheral by exposing vendor-specific
+GATT services.Currently only button notification and LED service are
+implemented.Other BLE sensor services can easily be added.
+Check `BlueST protocol`_ document.
+Although this sample can work with non stm32wb55 hardware it was tested
+only on nrf52840 module.
+
+Instructions for `ST Nucleo WB55RG`_ board:
+-------------------------------------------
+
+This sample requests having the stm32wb5x_BLE_Stack_fw.bin binary flashed
```

Comment:
I think "requests" should be "requires".  Where does someone get this ``stm32wb5x_BLE_Stack_fw.bin`` binary file?

Reply:
fixed